crates/ide/src/inlay_hints/param_name.rs

```diff
@@ -121,7 +130,9 @@ fn should_hide_param_name_hint(
     }
 
     let fn_name = match callable.kind() {
-        hir::CallableKind::Function(it) => Some(it.name(sema.db).display_no_db().to_smolstr()),
+        hir::CallableKind::Function(it) => {
+            Some(it.name(sema.db).display_no_db(edition).to_smolstr())
+        }
```

Comment:
I think here we want to always consider the non-r#-prefixed name?

Reply:
Either is fine, but I agree this makes more sense.